hail/python/test/hail/methods/test_simulation.py

```diff
@@ -8,10 +8,14 @@ def test_mating_simulation():
 
     n_samples = mt.count_cols()
 
-    assert hl.simulate_random_mating(mt, n_rounds=1, generation_size_multiplier=2, keep_founders=False).count_cols() == n_samples * 2
-    assert hl.simulate_random_mating(mt, n_rounds=4, generation_size_multiplier=2, keep_founders=False).count_cols() == n_samples * 16
-    assert hl.simulate_random_mating(mt, n_rounds=2, generation_size_multiplier=1, keep_founders=False).count_cols() == n_samples
-    assert hl.simulate_random_mating(mt, n_rounds=2, generation_size_multiplier=2, keep_founders=True).count_cols() == n_samples * 9
+    assert hl.simulate_random_mating(mt, n_rounds=1, pairs_per_generation_multiplier=0.5,
+                                     children_per_pair=2).count_cols() == n_samples * 2
+    assert hl.simulate_random_mating(mt, n_rounds=4, pairs_per_generation_multiplier=0.5,
+                                     children_per_pair=2).count_cols() == n_samples * 5
+    assert hl.simulate_random_mating(mt, n_rounds=3, pairs_per_generation_multiplier=1,
+                                     children_per_pair=2).count_cols() == n_samples + n_samples * 2 + n_samples * 4 + n_samples * 8
+    assert hl.simulate_random_mating(mt, n_rounds=2, pairs_per_generation_multiplier=0.5,
+                                     children_per_pair=1).count_cols() == n_samples * 1.75
 
-
-    hl.simulate_random_mating(mt, n_rounds=2, generation_size_multiplier=0.5, keep_founders=True)._force_count_rows()
\ No newline at end of file
+    hl.simulate_random_mating(mt, n_rounds=2, pairs_per_generation_multiplier=0.5,
+                              children_per_pair=2)._force_count_rows()
```

Comment:
Can we add a test where we simulate two rounds with just a few people and manually verify the relatednesses?